src/templates/app/cashier/account_transfer.jsx

```diff
@@ -12,68 +12,72 @@ const Row = ({ id }) => (
+                <p>
+                    <a href='javascript:;' id='reset_transfer'>{it.L('Make another transfer')}</a>
+                </p>
+            </div>
 
-            <Fieldset legend={it.L('Details')}>
-                <FormRow label={it.L('Transfer from')} type='label'  id='lbl_transfer_from' />
-                <FormRow label={it.L('Transfer to')}   type='select' id='transfer_to' />
-                <FormRow label={it.L('Amount')}        type='custom' id='transfer_amount'>
-                    <label id='currency' />
-                    <input id='amount' name='amount' type='text' maxLength='20' autoComplete='off' />
-                    <div className='hint' id='range_hint' />
-                </FormRow>
-            </Fieldset>
+            <form className='invisible' id='frm_account_transfer'>
+                <p>{it.L('Transfer funds between your real money accounts.')}</p>
 
-            <SubmitButton msg_id='form_error' type='submit' text={it.L('Transfer')} />
-        </form>
+                <Fieldset legend={it.L('Details')}>
+                    <FormRow label={it.L('Transfer from')} type='label'  id='lbl_transfer_from' />
+                    <FormRow label={it.L('Transfer to')}   type='select' id='transfer_to' />
+                    <FormRow label={it.L('Amount')}        type='custom' id='transfer_amount'>
+                        <label id='currency' />
+                        <input id='amount' name='amount' type='text' maxLength='20' autoComplete='off' />
+                        <div className='hint' id='range_hint' />
+                    </FormRow>
+                </Fieldset>
 
-        <div className='hint invisible' id='transfer_info'>
-            <p>{it.L('Note: Transfer between accounts is not available on weekends.')}</p>
-        </div>
+                <SubmitButton msg_id='form_error' type='submit' text={it.L('Transfer')} />
+            </form>
 
-        <div className='hint invisible' id='transfer_fee'>
-            {it.L('Notes:')}
-            <ul className='bullet'>
-                <li>{it.L('Transfer between accounts is not available on weekends')}</li>
-                <li>{it.L('You may only transfer funds between a fiat account and a cryptocurrency account')}</li>
-                <li>{it.L('Each transfer is subject to a [_1] transfer fee or a minimum fee of [_2], whichever is higher.', '<span id="transfer_fee_amount"></span>', '<span id="transfer_fee_minimum"></span>')}</li>
-                <li>{it.L('Authorised payment agents are exempted from paying any transfer fees')}</li>
-            </ul>
-        </div>
-    </React.Fragment>
-);
+            <div className='hint invisible' id='transfer_info'>
+                <p>{it.L(`Note: ${account_transfer_note}`)}</p>
+            </div>
+
+            <div className='hint invisible' id='transfer_fee'>
+                {it.L('Notes:')}
+                <ul className='bullet'>
+                    <li>{it.L(account_transfer_note)}</li>
```

Comment:
translation doesn't work this way, it needs to be wrapped around the string

Reply:
so just put the `account_transfer_note` in template strings? Like so? ```it.L(`${account_transfer_note}`)```

Reply:
Please note updated code.

Reply:
looks fine now, thanks